api_tests/e2e/rfc003/btc_eth/inverted_refund.ts

```diff
@@ -1,122 +1,46 @@
-import { expect } from "chai";
-import "chai/register-should";
-import { ethers } from "ethers";
-import { Actor } from "../../../lib/actor";
-import * as bitcoin from "../../../lib/bitcoin";
-import { ActionKind, SwapRequest } from "../../../lib/comit";
-import "../../../lib/setup_chai";
-import { createTests, Step } from "../../../lib/test_creator";
-import { HarnessGlobal } from "../../../lib/util";
-
-declare var global: HarnessGlobal;
-
-(async function() {
-    const alice = new Actor("alice", {
-        ledgerConfig: global.ledgerConfigs,
-        addressForIncomingBitcoinPayments:
-            "bcrt1qs2aderg3whgu0m8uadn6dwxjf7j3wx97kk2qqtrum89pmfcxknhsf89pj0",
-    });
-    const bob = new Actor("bob", {
-        ledgerConfig: global.ledgerConfigs,
-        addressForIncomingBitcoinPayments: null,
+import { Actor } from "../../../lib_sdk/actors/actor";
+import { AssetKind } from "../../../lib_sdk/asset";
+import { createActors } from "../../../lib_sdk/create_actors";
+
+setTimeout(function() {
+    let alice: Actor;
+    let bob: Actor;
+
+    beforeEach(async function() {
+        this.timeout(20000);
+        const actors = await createActors(
+            "e2e-rfc003-btc-eth-inverted-refund.log"
+        );
+        alice = actors.alice;
+        bob = actors.bob;
     });
 
-    const alphaAssetQuantity = 100000000;
-    const betaAssetQuantity = ethers.utils.parseEther("10");
-
-    const alphaExpiry = Math.round(Date.now() / 1000) + 13;
-    const betaExpiry = Math.round(Date.now() / 1000) + 8;
+    afterEach(() => {
+        if (alice) {
+            alice.stop();
+        }
+        if (bob) {
+            bob.stop();
+        }
+    });
 
-    await bitcoin.ensureFunding();
-    await bob.wallet.eth().fund("11");
-    await alice.wallet.eth().fund("0.1");
-    await alice.wallet.btc().fund(10);
+    describe("Alice can refund before Bob", function() {
```

Comment:
```suggestion
    describe("Alice refunds then Bob refunds", function() {
```
I am not sure how meaningful is `before Bob` part of the name as expiries are not taken in account in this test.